playwright/e2e/create-colony.spec.ts

```diff
@@ -0,0 +1,388 @@
+    });
+
+    test('Token Logo file uploader should work correctly', async ({ page }) => {
+      const validFilePath =
+        'playwright/fixtures/images/jaya-the-beast_400KB.png';
+
+      const invalidImage =
+        'playwright/fixtures/images/cat-image-new-1_3_MB.png';
+
+      // Select New token option
+      await page.getByLabel(/Create a new token/i).check();
+      await page.getByRole('button', { name: /continue/i }).click();
+
+      // Upload valid image
+      const tokenImageInput = page.locator('input[type="file"]');
+      await tokenImageInput.setInputFiles(validFilePath);
+
+      await expect(
+        page.getByRole('button', { name: /change logo/i }),
+      ).toBeVisible();
+
+      const tokenLogo = page.getByRole('img', { name: /avatar of token/i });
+      await expect(tokenLogo).toBeVisible();
+
+      // Removing the image
+      await page.getByRole('button', { name: /remove/i }).click();
+      await expect(tokenLogo).toBeHidden();
+
+      // Uploading invalid image
+      await tokenImageInput.setInputFiles(invalidImage);
+
+      // Validation message appears
+      const message = page.getByText(
+        /File size is too large, it should not exceed 1MB/i,
+      );
+      await expect(message).toBeVisible();
+      await expect(
+        page.getByRole('button', { name: /try again/i }),
+      ).toBeEnabled();
+
+      await page.getByTestId('file-remove').click();
+
+      await expect(message).toBeHidden();
+      await expect(page.getByTestId('file-remove')).toBeHidden();
+      await expect(
+        page.getByRole('button', { name: /try again/i }),
+      ).toBeHidden();
+    });
+
+    test('Should accept existing token and display validation message', async ({
+      page,
+    }) => {
+      await page.getByLabel(/Use an existing token/i).check();
+      await page.getByRole('button', { name: /continue/i }).click();
+
+      await expect(page.getByLabel(/Use an existing token/)).toBeChecked();
+
+      await expect(page.getByLabel(/Create a new token/i)).not.toBeChecked();
+      const input = page.getByLabel(/Existing token address/i);
+      await input.fill(invalidToken);
+
+      await expect(page.getByText(/Invalid address/i)).toBeVisible();
+      await expect(
+        page.getByRole('button', { name: /continue/i }),
+      ).toBeDisabled();
+      await input.fill(validToken);
+      await expect(page.getByText(/Invalid address/i)).toBeHidden();
+      await expect(page.getByText(/Token found:/i)).toBeVisible();
+      await expect(
+        page.getByRole('button', { name: /continue/i }),
+      ).toBeEnabled();
+    });
+  });
+
+  test.describe('Confirmation step', () => {
+    test.beforeEach(async ({ page }) => {
+      await fillColonyNameStep(page, {
+        nameFieldValue: colonyName,
+        urlFieldValue: customColonyURL,
+      });
+    });
+
+    test('Should render Confirmation step as expected', async ({ page }) => {
```

Comment:
This step fails again because of the token that doesn't exist

![Screenshot from 2024-10-17 17-59-41](https://github.com/user-attachments/assets/32c4c5af-ba77-412f-9b13-025452dd9192)

Reply:
updated, removed hardcoding the token address